### ISSUE TITLE
Looping an existing sound (possibly out of scope)

### ISSUE THREAD

**Issue by faijeya** [haoheliu/AudioLDM#59]
Hi.

Played a bit with the model (both -s models fit into 8GB VRAM, very nice) and tried my luck with SD-derived prompting skills.
Very impressive. Thank you very much for giving it to us.

I've got a weird request, I guess, or asking for guidance how to accomplish it.
I noticed that transfer with transfer strength 0 is okay-ish close to the original, despite it being decomposed to the latent space and then assembled back.

Is it possible to make or where should I start digging on my own to make a mode that will attempt to seamlessly loop an existing SFX snippet?

Making a seamless loop is usually manual work.
I'm aware only of nvk_LOOPMAKER plugin for Reaper that semi-automates the process and I kinda want to make a batch of them and go drink some tea.

Basically I just want to select a bunch of sounds from a soundbank (all licensed for commercial use, obviously), run a batch file, go drink some tea and return to processed loops that are close to the original and are looped seamlessly.

Maybe, hopefully, there's a magic prompt that will do it for me already? Fat chance, but you never know.

Thank you.
